contrib/debug/actions/print_ctx.py

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/python
+
+# Copyright 2020 The StackStorm Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import sys
+
+try:
+    from st2common.runners.base_action import Action
+except ImportError:
+    Action = object
+
+
+class PrintCtxAction(Action):
+    def run(self, ctx=None, *args, **kwargs):
+        print(ctx)
```

Comment:
Should we prettyprint it for nested objects?

Reply:
That's a good idea, but no. I want this action to do the absolute bare minimum so you know exactly what data you are feeding it.

That's a great idea for a different action though!